harness_scripts/functional_by_framework/check_by_framework_run1_step_entrypoint.sh

```diff
@@ -1,22 +1,25 @@
-#!/bin/bash
+#!/bin/sh
 
 set -e
+ROOT_DIR="$(pwd)"
 
-script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+script_dir="$(cd "$(dirname "$0")" && pwd)"
 . ${script_dir}/../common/common.sh
 
 FRAMEWORK=$1
-[ -z $FRAMEWORK ] && echo "Environment variable 'FRAMEWORK' is missing" && exit 1
+ENV_FOLDER=$2
+[ -z ${FRAMEWORK} ] && echo "Environment variable 'FRAMEWORK' is missing" && exit 1
+[ -z ${ENV_FOLDER} ] && echo "Environment variable 'ENV_FOLDER' is missing" && exit 1
 
-title "Assuming running integration tests in framework container (inside Docker), for env: '$FRAMEWORK'"
+title "Assuming running integration tests in framework container (inside Docker), for env: '${ENV_FOLDER}/${FRAMEWORK}'"
 
-ROOT_DIR="$(pwd)"
-PUBLIC_ENVS_DIR="${ROOT_DIR}/public_dropin_environments"
-REQ_FILE_PATH="${PUBLIC_ENVS_DIR}/${FRAMEWORK}/requirements.txt"
-[ ! -f $REQ_FILE_PATH ] && echo "Requirements file not found: $REQ_FILE_PATH" && exit 1
 
-script_dir=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
-. ${script_dir}/../../tools/update-python-to-meet-requirements.sh
+PUBLIC_ENVS_DIR="${ROOT_DIR}/${ENV_FOLDER}"
+ENV_REQ_FILE_PATH="${PUBLIC_ENVS_DIR}/${FRAMEWORK}/requirements.txt"
+[ ! -f ${ENV_REQ_FILE_PATH} ] && echo "Requirements file not found: ${ENV_REQ_FILE_PATH}" && exit 1
+
+# All our envs have python>3.9, comment out for now
```

Comment:
This script uses more commands that don't exist in the image.
So i commented it out because of that and that all our envs are >= Py 3.9